files/management/commands/fix_media_paths.py

```
@@ -0,0 +1,236 @@
+                if new_path != old_path:
+                    if dry_run:
+                        self.stdout.write(
+                            self.style.WARNING(f'  Would fix Media {media.pk}: {old_path} → {new_path}')
+                        )
+                        fixed_count += 1
+                    else:
+                        try:
+                            media.preview_file_path = new_path
+                            media.save(update_fields=['preview_file_path'])
+                            fixed_count += 1
+                        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)